pkg/csi/cinder/openstack/openstack_volumes.go

```diff
@@ -396,6 +397,21 @@ func (os *OpenStack) GetMaxVolLimit() int64 {
 	return defaultMaxVolAttachLimit
 }
 
+// GetFreeCapacity returns free capacity of the block storage, in GB
```

Comment:
at least this is not my understanding 
the free storage usually comes from backend and I don't know whether every storage provider has this 

and the API seems from here https://github.com/openstack/cinder/blob/master/cinder/api/v3/limits.py#L25 ? 
if so ,it's actually the quota, not the real storage ? e.g you can have 100G max NFS pool
but you can set quota to 1T  ?

Reply:
Yep, this is a quota.
Based on this quota, I cannot allocate more disk space as I am not the owner of the OpenStack cluster.

These metrics provide the scheduler with additional information about which availability zone the pod can be scheduled in.

Reply:
um... at least the CSI GetFreeSpace is expecting to give `real` free space? 
this might confuse a little bit

and if the quota is bigger than real storage, this is not accurate info?

Reply:
Oh, now I see that you mean. thanks.
I've tried to get the real size of the pool, but it required `rule:admin_api` role for the api method [scheduler_extension:scheduler_stats:get_pools](https://docs.openstack.org/cinder/latest/configuration/block-storage/policy.html)

So, It is better to rename the function to GetFreeQuotaSpace since we cannot get the real size of the pool.